Roslyn.sln

```diff
@@ -5692,6 +5696,86 @@ Global
 		{DEB3D675-5A3C-46DA-8945-F2EFAB135EA0}.Release|x64.Build.0 = Release|Any CPU
 		{DEB3D675-5A3C-46DA-8945-F2EFAB135EA0}.Release|x86.ActiveCfg = Release|Any CPU
 		{DEB3D675-5A3C-46DA-8945-F2EFAB135EA0}.Release|x86.Build.0 = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.Debug|ARM.ActiveCfg = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.Debug|ARM.Build.0 = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.Debug|Mixed Platforms.ActiveCfg = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.Debug|Mixed Platforms.Build.0 = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.Debug|x64.Build.0 = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.Debug|x86.Build.0 = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodDebug|Any CPU.ActiveCfg = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodDebug|Any CPU.Build.0 = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodDebug|ARM.ActiveCfg = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodDebug|ARM.Build.0 = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodDebug|Mixed Platforms.ActiveCfg = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodDebug|Mixed Platforms.Build.0 = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodDebug|x64.ActiveCfg = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodDebug|x64.Build.0 = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodDebug|x86.ActiveCfg = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodDebug|x86.Build.0 = Debug|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|Any CPU.ActiveCfg = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|Any CPU.Build.0 = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|ARM.ActiveCfg = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|ARM.Build.0 = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|Mixed Platforms.ActiveCfg = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|Mixed Platforms.Build.0 = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|x64.ActiveCfg = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|x64.Build.0 = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|x86.ActiveCfg = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|x86.Build.0 = Release|Any CPU
```

Comment:
The DogfoodDebug and DogfoodRelease entries look new.  Is there a reason that we added them?